cmd/utils/flags.go

```diff
@@ -192,7 +192,11 @@ var (
 	}
 	DeveloperPeriodFlag = cli.IntFlag{
 		Name:  "dev.period",
-		Usage: "Block period to use in developer mode (0 = mine only if transaction pending)",
+		Usage: "Block period for proof-of-authority network to use in developer mode (0 = mine only if transaction pending)",
+	}
+	DeveloperPoWFlag = cli.BoolFlag{
+		Name:  "dev.pow",
+		Usage: "Ephemeral proof-of-work network with a pre-funded developer account, mining enabled",
```

Comment:
We need an incompatibility check (with `--dev` and `--dev.period`) for this new flag. `utils` has a helper function for that somewhere.

Reply:
`CheckExclusive`

Reply:
I've done this already (locally) :)